use_case_examples/lora_finetuning/GPT2FineTuneHybrid.ipynb

```diff
@@ -846,7 +869,7 @@
     "tokenizer.parallelism = False\n",
     "\n",
     "# Train the model using FHE simulation\n",
-    "train_custom_model(hybrid_model, train_dataloader, training_args, tokenizer, fhe=\"simulate\")"
+    "train_custom_model(hybrid_model, train_dataloader, training_args, tokenizer, fhe=\"disable\")"
    ]
   },
   {
```

Comment:
After the loss plot, there is a typo

I suggest:
Note that our goal is to showcase the use of FHE for encrypted fine-tuning. The dataset consists of 68 examples and a total of 2,386 tokens, which is relatively small. Despite its limited size, which offers little support for the model's learning process, it still manages to produce interesting results.

Reply:
Alright I did the change